client/test-data/bazaar/technopark/offerings/EdisonKaffee.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
on folder naming:

please replace "technopark" with "EdisonPaula"
subfolders "businesses / offerings do not make sense for me as it isn't clear if you want to group by business and its offerings, or as you did. I suggest to reflect `business` and `product` in filenames to reflect the type

Reply:
i now created a folder structure of test-data/bazaar/business/products, ok?